pages/index.js

```diff
@@ -9,13 +10,14 @@ export default function Home() {
   const [jobs, setJobs] = useState([]);
   const [rows, setRows] = useState([]);
   const [expandedRows, setExpandedRows] = useState([]);
+  const [requiredFilter, setRequiredFilter] = useState(false);
 
   useEffect(() => {
     const fetchData = async () => {
       let data = {};
 
       if (process.env.NODE_ENV === "development") {
-        data = (await import("../job_stats.json")).default;
+        data = (await import("../data/job_stats.json")).default;
```

Comment:
Please leave this alone as per #20 

```suggestion
        data = (await import("../job_stats.json")).default;
```